MantidPlot/src/Plot3DDialog.cpp

```diff
@@ -783,7 +783,7 @@ bool Plot3DDialog::updatePlot()
 
 		try {
 			MyParser parser;
-			parser.SetExpr(to.ascii());
+			parser.SetExpr(to.latin1());
```

Comment:
Is there a reason you are using `.latin1`? shouldn't this be be `toAscii()` or `.toAscii().constData()` as you can find [here](http://doc.qt.io/qt-4.8/qstring-compat.html#ascii) ..